app/src/models/services/Student/Student.js

```diff
@@ -18,44 +18,75 @@ class Student {
     this.SALT_ROUNDS = Number(process.env.SALT_ROUNDS);
   }
 
-  async login() {
+  static makeResponseMsg(status, msg, jwt) {
+    return {
+      success: status < 400,
+      status,
+      msg,
+      jwt,
+    };
+  }
+
+  // static successMessage(msg, jwt) {
+  //   return {
+  //     success: true,
+  //     msg,
+  //     jwt,
+  //   };
+  // }
+
+  // static failMessage(msg, status) {
+  //   return {
+  //     success: false,
+  //     msg,
+  //     status,
+  //   };
+  // }
+
+  blankInputCheck() {
     const client = this.body;
 
     if (!(client.id && client.password)) {
-      return { success: false, msg: '아이디 또는 비밀번호를 확인해주세요.' };
+      return Student.makeResponseMsg(
+        400,
+        '아이디 또는 비밀번호를 확인해주세요.'
+      );
     }
+    return { success: true };
```

Comment:
```suggestion
    return;
```
성공과 실패 여부를 msg 유무로 판단하므로 return 값을 주지 않아도 괜찮을 것 같아요